src/IndividualReview.jsx

```diff
@@ -1,24 +1,29 @@
 import { useEffect, useState } from "react";
-import axios from "axios";
-import { useLocation, useParams } from "react-router";
+import { useParams } from "react-router";
 import { Link } from "react-router-dom";
+import { getReviewById } from "./Api";
+import { formatDate } from "./utilities";
 
 const IndividualReview = () => {
   let { reviewId } = useParams();
-  const [review, setReview] = useState("");
+  const [review, setReview] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [date, setDate] = useState("");
 
   useEffect(() => {
-    axios
-      .get(
-        `https://fame-boardgame-review-website.onrender.com/api/reviews/${reviewId}`
-      )
-      .then(({ data }) => {
-        setReview(data.review);
-      });
+    setIsLoading(true);
+    getReviewById({ setReview, reviewId });
+    setIsLoading(false);
   }, []);
 
+  // useEffect(()=>{
+  //   console.log(Date.parse(review.created_at));
+  //   // setDate(formatDate(Date.parse(review.created_at)));
+  //   console.log(date);
+  // }, review)
+
   return (
```

Comment:
We should make use of our loading state here and display a loading message if we're still waiting for our reviews to be retrieved. Otherwise, we can go ahead and display the reviews